UI/RemoteISOScreen.cpp

```diff
@@ -598,8 +600,8 @@ RemoteISOBrowseScreen::RemoteISOBrowseScreen(const std::vector<std::string> &gam
 void RemoteISOBrowseScreen::CreateViews() {
 	bool vertical = UseVerticalLayout();
 
-	I18NCategory *mm = GetI18NCategory("MainMenu");
 	I18NCategory *di = GetI18NCategory("Dialog");
+	I18NCategory *n = GetI18NCategory("Networking");
```

Comment:
Ah...
I used "Networking" as a general purpose network-related category.
Though I do understand the difference between the functionality you explain and netplay.

Therefore I think we should make the distinction more obvious.
Why not having 2 categories ? :
 - "Networking" --> remote ISO feature
 - "NetPlay"/"Adhoc"/"Multiplayer" (which name ?) --> netplay